lib/grape_log_formatter/formatters/custom_format.rb

```diff
@@ -6,7 +6,13 @@ class CustomFormat < GrapeLogging::Formatters::Lograge
       def call(severity, datetime, _, data)
         lograge_format = super
 
-        "[#{datetime.strftime('%Y-%m-%d %H:%M:%S')}] #{severity} -- #{lograge_format}\n"
+        if defined?(Datadog::Tracing.log_correlation)
+          dd_format = "[#{Datadog::Tracing.log_correlation}] "
```

Comment:
do we need a space in the string end?

Reply:
Shoot. I swapped the order so it's supposed to be at the beginning now. But yes it was intentional.